src/service/url-replacements-impl.js

```diff
@@ -812,6 +813,30 @@ export class UrlReplacements {
   }
 
   /**
+   * Append URL parameters from data-amp-addparams to URL
+   * @param {string} url
+   * @param {!Element} element An anchor element.
+   * @return {string} url
+   * @private
+   */
+  extendURLParams_(url, element) {
+    const additionalURLParamaters = element.getAttribute('data-amp-addparams');
```

Comment:
- I can make it as `data-vars-urlparam` from `data-amp-addparam` but it wont align with naming of `data-amp-replace`. Is that fine ? 
- Also `data-amp-addparam` can have token values like `c=CLIENT_ID(abc)` which should be replaced by entries provided on `data-amp-replace` like 
```
a.setAttribute('data-amp-replace', 'QUERY_PARAM CLIENT_ID');
a.setAttribute('data-amp-addparams', 'guid=123 c=CLIENT_ID(abc)');
```
In this if we have a different data attribute name like `data-vars-*` I guess it might create confusion . Please let me know your suggestion on this